cmd/kube-proxy/app/server.go

```diff
@@ -477,6 +477,11 @@ func NewProxyServer(config *componentconfig.KubeProxyConfiguration, cleanupAndEx
 			return nil, fmt.Errorf("unable to read IPTables MasqueradeBit from config")
 		}
 
+		nodeIP, err := getNodeIP(client, hostname)
+		if err != nil {
+			return nil, fmt.Errorf("unable to fetch node IP: %v", err)
```

Comment:
Oh, I see.  Is there a clear way for the proxy to tell whether it is intended to be running standalone?